google/cloud/firestore_v1/_helpers.py

```diff
@@ -921,7 +926,7 @@ class ReadAfterWriteError(Exception):
     """
 
 
-def get_transaction_id(transaction, read_operation=True) -> Any:
+def get_transaction_id(transaction, read_operation=True) -> Optional[bytes]:
```

Comment:
I didn't think `Optional` was appropriate for return values, but according to [the docs](https://docs.python.org/3/library/typing.html#typing.Optional), it seems like it might be so.  It would, however, seem clearer as:
```suggestion
def get_transaction_id(transaction, read_operation=True) -> Union[bytes, None]:
```